selector/src/shared/notebook-tags.js

```diff
@@ -17,6 +17,7 @@ export const TASKS = /** @type {const} */ ({
     TEXT_TO_VIDEO: 'Text-to-Video',
     VIDEO_TO_TEXT: 'Video-to-Text',
     TEXT_TO_AUDIO: 'Text-to-Audio',
+    TEXT_TO_SPEECH: 'Text-to-Speech',
```

Comment:
I believe TEXT_TO_AUDIO category is the same what is TEXT_TO_SPEECH, just generalized, for generation audio like riffusion used AUDIO_GENERATION

Reply:
Agree. But I thought that we have the general type TTS in the model name, but we haven't that type for searching on github.io. HF for example has both. So, I assigned both types for this model only for search purpose.

Reply:
then please add Text-to-Speech to bark too (maybe later) for alignment